monitoring/router.nginx.conf

```diff
@@ -1,20 +1,45 @@
 server {
-    server_name _;
+    server_name grafana.*;
 
     location /monitoring/grafana/ {
         proxy_pass http://grafana/;
-        proxy_set_header Host internal.hail.is;
+        proxy_set_header Host $host;
```

Comment:
I'm not sure why you changed this, it might be part of the problems you're seeing.  If memory serves, these services use Host in various redirects, so it needs to be host that originally came from the user.  That's `internal.hail.is`.

Reply:
Grafana doesn't actually seem to care. I tried both internal.hail.is in monitoring.yaml, and grafana.hail.is, both worked. So either it isn't being used, or both get routed correctly, when host is $host.